### DISCUSSION TITLE
GitHub Enterprise Server on AWS stuck at "Migration"

### DISCUSSION THREAD

**Discussion by muditpathak08** [community/community#81449] [Enterprise]
### Select Topic Area

Question

### Body

We are hosting a GitHub Enterprise Trial License on EC2 r6x large instance. We have taken care of the Minimum Hardware requirements and imported the License but after accessing the DNS page ofthe EC2 it gets stuck at this page and crashes after 24 hours.

Below is the Page where is gets tuck for almost 24 hours and then crashes later on and the page gets inaccessible
Please suggest
![GitHub_Enterprise_Data_Migration](https://github.com/community/community/assets/67001611/95f193a0-7e60-4522-9173-cf46df40955e)

**Comment by tbakar10** [comment]
Hi @muditpathak08 , did you manage to get it resolved? If so, how?
